pkg/controller/hosted/controller.go

```diff
@@ -300,7 +300,8 @@ func createHostedManifestWork(managedClusterName string,
 			Name:      hostedKlusterletManifestWorkName(managedClusterName),
 			Namespace: manifestWorkNamespace,
 			Labels: map[string]string{
-				constants.HostedWorksLabel: "true",
+				constants.HostedWorksLabel:   "true",
```

Comment:
do we still need this label?

Reply:
We need this, we use label to informer hosted works https://github.com/stolostron/managedcluster-import-controller/blob/main/cmd/manager/main.go#L171 to reduce the cache

Reply:
We can only check the hosted cluster label, will remove this